src/Command/DecodeCommand.php

```diff
@@ -26,6 +28,7 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $style = new SymfonyStyle($input, $output);
```

Comment:
You should use `Command::initialize` method to instantiate a new `SymfonyStyle`, that will be available for all methods of your command.

Reply:
I disagree. Initialize is not done for that (it's done to populate missing argument). And it'll add a state in a (service) class. It should be avoided